### ISSUE TITLE
Request new default keybind for crafting menu "hide"

### ISSUE THREAD

**Issue by curstwist** [CleverRaven/Cataclysm-DDA#26361]
**Describe the plea**  
Please reconsider the default keybind "s" for hiding items in the crafting menu.

Reasoning:
Very common key results in hiding recipes unintentionally.  I often end up hiding items when I forget to bring up the search box and begin typing a s:[skill] search.  I'll unbind the key but that shouldn't be necessary.  Also the lack of warning/transparency can be another barrier for new players.

**Expected behavior**  
Common keys shouldn't be used to hide content.

**Versions and configuration(please complete the following information):**
 - OS:Windows 10 

 - Graphics version Tiles

**Comment by denemcam** [comment]
+ for that, accidently pressed it when first time entered the game, and was wondering where the craftable recipes gone

**Comment by AMurkin** [comment]
Search for `h:y` and unhide them.

**Comment by chrisneave** [comment]
> Search for `h:y` and unhide them.

I had the same issue but this suggestion didn't work for me. I also tried a search string of `h:yes` but still no joy. I'm running build 9307 with no mods.

If someone else hits this issue and can't use the filter to show the hidden recipes then here's how I fixed it:

1. Close CDDA if its running.
2. Open the `save/<your_world>/uistate.json`.
3. Remove all items from the `"hidden_recipes"` property so that it looks like this: `"hidden_recipes":[]`.
4. Start CDDA and reload your game.
